src/prefect/serialization/state.py

```diff
@@ -68,6 +68,8 @@ class QueuedSchema(MetaStateSchema):
     class Meta:
         object_class = state.Queued
 
+    start_time = fields.DateTime(allow_none=True)
```

Comment:
So your serialization tests are failing because of this --> marshmallow assumes that any attributes in your schema can be provided to `__init__` to recreate the object.  To get around this, you need to implement a method on this `QueuedSchema` class like so:
```python
@post_load
def create_object(self, data: dict) -> state.State:
    start_time = data.pop("start_time", None)
    base_obj = super().create_object(data)
    base_obj.start_time = start_time
    return base_obj
```
let me know if that makes sense or if you have any questions!

Reply:
That makes complete sense-- thank you, Chris!